reconcile/utils/oc.py

```diff
@@ -971,21 +987,20 @@ def __init__(
         if server:
             self.client = self._get_client(server, token)
             self.api_kind_version = self.get_api_resources()
+            self.api_resources = self.api_kind_version.keys()
+            self.init_api_resources = True
```

Comment:
Would it make sense to move that within `self.get_api_resources()` ?

Reply:
I would not like to do that, get_api_resources() is only resolving values for  `api_kind_version`. The naming is messed up here. I'd rather change openshiftresources then.